packages/naming/src/tests/rewrite.test.ts

```
@@ -1,5 +1,5 @@
 import test from "ava";
-import { mkAliasRewriter, mkRelativeToJs } from "../lib.js";
+import { mkAliasRewriter, mkRelativeToJs } from "../rewrite.js";
```

ESLint

[error] There should be at least one empty line between import groups

(import/order)